package.json

```diff
@@ -29,22 +27,14 @@
     "url": "https://github.com/seegno/eslint-config-seegno/issues"
   },
   "homepage": "https://github.com/seegno/eslint-config-seegno#readme",
-  "dependencies": {
-    "eslint": "^1.8.0"
-  },
   "devDependencies": {
-    "babel": "^5.8.23",
     "babel-eslint": "^4.1.3",
+    "eslint": "^1.8.0",
     "eslint-plugin-babel": "^2.1.1",
     "mocha": "^2.3.3",
-    "should": "^7.1.0"
-  },
-  "peerDependencies": {
-    "babel-eslint": ">= 4.1",
-    "eslint": ">= 1.8",
-    "eslint-plugin-babel": ">= 2.1"
```

Comment:
Same argument as on the jscs preset project. One of the most prolific eslint presets also decided to avoid peerDependencies' hell - https://github.com/yannickcr/eslint-plugin-react/blob/master/package.json. There's enough evidence it *will* turn into a problem.